### PR TITLE
i18n(es): Translate `index.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Translated `index.mdx` to spanish

### CODE REVIEW

File: docs/src/content/docs/es/index.mdx [hunk 1]
```diff
@@ -1,26 +1,43 @@
...
+  <Card title="Documentación que deleita" icon="open-book">
+    Incluye: navegación del sitio, búsqueda, internacionalización, SEO, tipografía fácil de leer, resaltado de código, modo oscuro y mucho más.
+  </Card>
+  <Card title="Impulsado por Astro" icon="rocket">
+    Aprovecha todo el poder y rendimiento de Astro. Amplía Starlight con tus integraciones y bibliotecas favoritas de Astro.
+  </Card>
+  <Card title="Markdown, Markdoc y MDX" icon="document">
+    Lleva tu lenguaje de marcado favorito. Starlight te ofrece validación de frontmatter integrada con seguridad de tipos en TypeScript.
+  </Card>
+  <Card title="Lleva tus propios componentes de de UI" icon="puzzle">
```
Comment (delucis) [inline docs/src/content/docs/es/index.mdx:33]: There should only be one "de" here right?
```suggestion
  <Card title="Lleva tus propios componentes de UI" icon="puzzle">
```

**Review by delucis — COMMENTED** [review]
Amazing, thank you @Waxer59! Just spotted one thing.

File: docs/src/content/docs/es/index.mdx [hunk 2]
```diff
@@ -1,26 +1,43 @@
...
+hero:
+  title: Haz que tus docs brillen con Starlight
+  tagline: Todo lo que necesitas para construir un sitio web de documentación estelar. Rápido, accesible y fácil de usar.
+  image:
+    file: ../../../assets/hero-star.webp
+  actions:
+    - text: Empezar
+      icon: right-arrow
+      variant: primary
+      link: /getting-started/
```
Comment (morinokami) [inline docs/src/content/docs/es/index.mdx:14]: Hi, I just noticed something while casually browsing. Shouldn't the link start with `/es/`?

```suggestion
      link: /es/getting-started/
```

Comment (delucis) [reply docs/src/content/docs/es/index.mdx:14]: Oh yes! Very well spotted!

I don't have the same link checking set up here that we have in docs yet, so easy for us to miss this kind of thing. Thanks for looking!

**Pull request merged**